www/perl-lib/SMT/Rest/SCCAPIv1.pm

```diff
@@ -10,8 +10,7 @@ use JSON;
 
 use SMT::Utils;
 use SMT::Client;
-use SMT::Registration;
-
+use SMT;
```

Comment:
I havn't tested this yet, but I do not see a reason for changing the import and I fear that now calling
SMT::Registration::findCatalogs() cannot be executed anymore.

Was this change on purpose or by accident?